dht_test.go

```diff
@@ -1403,6 +1403,10 @@ func testFindPeerQuery(t *testing.T,
 		connect(t, ctx, guy, others[i])
 	}
 
+	for _, d := range dhts {
+		d.RefreshRoutingTable()
+	}
```

Comment:
@Stebalien this is sort of interesting in that it shows that our new logic works on valid routing tables and not invalid/random tables.